community/rds/aurora-serverless/template.yml

```diff
@@ -0,0 +1,70 @@
+---
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: Aurora serverless cluster
+
+Parameters:
+  DatabaseName:
+    Type: String
+  MasterUserPassword:
+    Type: AWS::SSM::Parameter::Value<String>
+    Description: Parameter Store name
+  VpcId:
+    Type: String
+  VpcSecurityGroupId:
```

Comment:
I'm having a little trouble following the networking in this example. The SG in the parameters is associated with the cluster, which is then associated with the "SecurityGroup" resource you define (which looks to be the bastion Security Group), however you end up assigning both to the bastion? Why does bastion need both groups?

Also, as with the VpcId above, this should be the AWS-Specific "AWS::EC2::SecurityGroup::Id" parameter type (see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html)

Reply:
If I remove the VPC security group from the EC2 instance I'm not able to access the database anymore.

Reply:
See, this is why I'm a CloudFormation DA, and not a networking DA. :)

Reply:
You're observation made sense. I tried that out but I wasn't able to connect to the database anymore. Maybe there is a better way for allowing the bastion to access the database but I honestly don't know it 😅